logic/linkPreviewLogic.js

```diff
@@ -107,15 +107,18 @@ module.exports = class LinkPreviewLogic {
   }
 
   static async createPreviewForUrl(url, crawler) {
+    // VERIFY URL
+    const urlProtocol = url.match(/^[^:]+(?=:\/\/)/);
+    const newUrl = !urlProtocol ? `http://${url}` : url;
```

Comment:
```suggestion
    const newUrl = !urlProtocol ? `https://${url}` : url;
```

Let's use secure connection by default? `https`

Reply:
I was thinking about this as well but since its a one-shot try I think http is the more common and therefore the better choice

Reply:
It will result in browser errors for serving insecure content though.

Reply:
why, we are not serving anything from the website.